src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -184,6 +189,18 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(Items.POISONOUS_POTATO).grist(AMBER, 4).grist(IODINE, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.MELON_SLICE).grist(AMBER, 1).grist(CAULK, 1).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.ENCHANTED_GOLDEN_APPLE).grist(AMBER, 4).grist(GOLD, 150).grist(URANIUM, 10).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.AZALEA).grist(MARBLE, 4).grist(IODINE, 6).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.AZALEA_LEAVES).grist(MARBLE, 4).grist(IODINE, 6).grist(URANIUM, 3).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.BIG_DRIPLEAF).grist(URANIUM, 2).grist(SHALE, 4).grist(MERCURY, 14).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.SMALL_DRIPLEAF).grist(URANIUM, 1).grist(SHALE, 2).grist(MERCURY, 6).buildFor(recipeSaver, Minestuck.MOD_ID);
```

Comment:
Not sure how would be best to handle this but since small dripleafs can be converted into a big dripleaf several blocks tall, they present an opportunity for grist duplication. Other plants also result in a gain in grist but consider that when deciding the costs

Reply:
lowered their worths by a bunch but it may be fine to let that slide.
being lazy but farming isn't a thing in the medium so having a good source of passive grincome could be pretty dope.
the idea here is setting up a farm, adventuring out, and coming home to reap the rewards

Reply:
Should be fine in the end but they do not grow on their own and can only grow via bonemealing